src/shared/components/download/__tests__/Download.spec.tsx

```diff
@@ -35,6 +35,10 @@ describe('Download component', () => {
   let onCloseMock;
 
   beforeEach(() => {
+    window.localStorage.setItem(
```

Comment:
Thinking out loud here, but wondering if this could be done in `customRender` and the `window.localStorage.clear();` triggered when the Wrapper component unmounts (through a `useEffect` return maybe?).

Reply:
`useEffect` wouldn't work because it happens after the first render. But I liked the idea and modified it. Basically this way the customRender keeps the same signature than before the refactor, but we set the localStorage in the Wrapper's constructor (yes, had to use classes, as there's no hook for logic _before_ the first render), and cleanup in `componentWillUnmount()`.
Now in latest commit

Reply:
Shame about the cleanup but it does look neater 👍

Reply:
Oh actually yeah cleanup is good too 😆